server/vulnerabilities/vuln_centos/centos.go

```diff
@@ -182,9 +185,7 @@ func ParseCentOSRepository(opts ...CentOSOption) (CentOSPkgSet, error) {
 			for cve := range cves {
 				cveList = append(cveList, cve)
 			}
-			if opts_.verbose {
```

Comment:
Any reason to always print all the pkgs and their cve list?

Reply:
See line 182. The previous version of this code had multiple `if opts_.verbose` checks that were redundant. I reduced it to one.